docs/wiki/Whats-new.md

```diff
@@ -2,22 +2,86 @@
 
 - [In this Section](#in-this-section)
 - [Updates](#updates)
+  - [November 2022](#november-2022)
+    - [Policy](#policy)
```

Comment:
Looks like new subheading have been dragged in by mistake. I suspect an extension has done this in VSCode

Reply:
Just months are in the table now